src/hooks/Authentication/index.js

```diff
@@ -51,6 +33,11 @@ export const useSignOut = () => {
   return () => signOut(context);
 };
 
+// Other hooks
+export { useAnswerChallenge } from './AnswerChallenge';
+export { useVerification } from './Verification';
+export { useLocalStorageUser } from './LocalStorageUser';
+
```

Comment:
Mostly because the `useAnswerChallenge` and `useVerification` are being used elsewhere and I wanted to break up the code without breaking the imports since those other hooks are used elsewhere.

Reply:
ah right. I would have changed the imports in question, but don't have a really strong option about it.

Reply:
I think the anonymized email would be a good idea for users without a username...

Reply:
Removing the user id in the local storage, if it is incorrect, would be cool actually.

Reply:
Hmh, about returning the user id after self-scanning: So far we have been trying to avoid an api which would map an email to a user id, so this would be a first. I wouldn't exactly call it a security issue, because you can't really do a whole lot with the userId (and without a token) which you wouldn't be able to do with just the email address anyway. 

I mean, our main reasoning for not mapping emails to user id was so possible attackers are not able to check if an email exists, but i guess you are able to do that anyway on mutliple occasions (e.g. when signing up through cognito, when self scanning an error is returned that a given mail does not exist, and so forth). 

In conclusion I would be open to it, if improves it UX.